itomate/itomate.py

```diff
@@ -95,7 +98,12 @@ async def render_tab_panes(tab, panes):
                 focus_session = current_session
 
             # Execute the commands for this pane
-            pane_commands = horizontal_pane.get('commands') or []
+            pane_commands = []
+
+            if root_path:
+                pane_commands.append(f"cd {root_path}")
+
+            pane_commands.extend(horizontal_pane.get('commands') or [])
```

Comment:
Thanks @JohnLegrandRichards for the PR. 
How about we do this part [before we pass the panes for rendering](https://github.com/kamranahmedse/itomate/blob/master/itomate/itomate.py#L148) so that this chunk is not duplicated in two places?

Reply:
@kamranahmedse I can see where I might be able to format the command before it gets passed f"cd {root_path}", but I am having a hard time seeing where I could create the commands array before it gets passed. In the vertical panes section I need to make sure we are not on the current open session before I append the root path, and I need to be inside the rendering loop in order to add the root command to the array of pane commands.

Reply:
> In the vertical panes section I need to make sure we are not on the current open session before I append the root path

[This shouldn't be the case](https://github.com/kamranahmedse/itomate/pull/7/files#diff-44fd2689e801cb8ec3bd23e3d5a10799R60-R61). We should actually append that to the currently open session's commands as well.

>  I would need to loop through all panes commands and add the root path before passing for rendering. Which seems like an unnecessary loop.

For the additional loop, I think it's fine in this case. Keeping in view the nature and use of the project I don't think it would be that much of an overhead. I would suggest prepending `f"cd {root_path}"` before we pass the panes for rendering.

Reply:
@kamranahmedse Ok I can add the loop before the rendering and prepend the root_path command.

As far as the current open session window.. This is my use case.. so I have the itomate.yml file in my projects 'documents/myproject' Therefore I will be running the Itomate command from that directory. Well I also need a bunch of other services spun up from within this project.. maybe a rails console and a sidekiq instance. If I prepend it to my current open session window it will just fail right? because I am already there.

Reply:
Perfect, let's do it and then get it merged :)

Regarding the currently open session, I don't think it will fail for you also because it is expected from you to provide the absolute path (e.g. `~/Workspace/my-project` for `root` so it doesn't matter where you are or where the itomate configuration is put.

Reply:
@kamranahmedse Change has been made. Ready to go